pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+Staff with the required permissions can ban a user directly from a report page, using the ‘Ban user’
+button.
+
+<img alt="Ban a user directly from a report page" src="/assets/img/pro-user-guide/ban-user-report-page.png" class="pro-screenshot" />
+
+#### Removing users from the banned list
+
+When a user has been banned, an ‘unban’ button will be visible on their records and reports. This
+restores them to the status of a standard user, but it does not have any effect on reports which
+were made during the period when the user was banned — these remain unsent.
+
+</div>
+
+<div class="pro-task" markdown="1" id="correct-reporter-errors">
+
+### Correcting reporter errors
+
+#### Changing a report’s category
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff, and ‘edit report category’ must be ticked.</span>
+
+If the user has chosen the wrong category for their report, when logged in, staff with the appropriate permissions can change it from the dropdown to the right of the report.
+
+<img alt="Change the category of a report" src="/assets/img/pro-user-guide/change-category-report.png" class="pro-screenshot" />
+
+If changing the category of the report results in it becoming the responsibility of another authority,
+the report will be re-sent to that authority. Do not mark it as closed, even though you have now
+dealt with it as far as your council is concerned; from the other authority's point of view, and for
+the person who made the report, the status is still open.
+
+However, if the report remains your council’s responsibility, the category change does not re-send
+the report unless the send method has changed (e.g. in the case of councils where some reportsare sent via the Open311 protocol and others by email).
+
+#### Changing the location
+Permissions required: User must be marked as staff, and ‘Edit report priority’ must be ticked.
+- You can click and drag a pin at any time to change the report’s specified location.
+- When viewing a report on the ground, staff with the appropriate permissions may click
+‘set to my current location’. This will reset the co-ordinates of the issue to reflect a more
+
+</div>
+
+<div class="pro-task" markdown="1" id="keep-report-maker-informed">
+
+### Keeping the report-maker informed
+
+Many issues require identical or very similar responses. Rather than type your reply out each time,
+staff can choose from a number of pre-written (but still editable) template responses, such as
+‘Thank you for your report, we’ll be back in touch with an update soon’, ‘Repairs are now
+underway’ and ‘This issue is now closed’.
+
+#### Status updates
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff</span>
+
+<img alt="Use a template to make public updates faster" src="/assets/img/pro-user-guide/report-public-update.png" class="pro-screenshot" />
+
+From the report page, staff with the appropriate permissions may select from the ‘public update’
```

Comment:
It's also above the normal update form too, which isn't labelled 'public update'

Reply:
Do you mean this box?
![screenshot-2018-5-24 concrete post broken - viewing a problem](https://user-images.githubusercontent.com/944656/40488364-82549366-5f5e-11e8-8aa2-e43a25ea63b2.png)

I'm a bit lost as to exactly what this text refers to - is it the selecting of a pre-written template response? Would staff have access to those templates via this box as well?

Reply:
Updated